main.yml

```diff
@@ -86,9 +82,16 @@
         and cache_machine.state|default(omit) == "present"
       tags: ['update','idc_import']
 
+    - role: logs
+      when: ansible_local.installed_software.management.managed_by_bsf|bool == True
+
```

Comment:
no tags, not sure this is what you wanted?

Reply:
You'll find them within the role